### PR TITLE
SQL: series tag NULL condition

### PR DESCRIPTION
**Pull request by unrealwork**
## SQL: series tag NULL condition
- [x] Tests

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/operator/SqlOperatorNotEqualsWithNullTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+     */
+    @Test
+    public void testIgnoringNullObjectsComparison() {
+        String sqlQuery = String.format(
+                "SELECT value FROM '%s'%nWHERE tags.a != 'b'",
+                TEST_METRIC_NAME
+        );
+
+        Response response = executeQuery(sqlQuery);
+
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/operator/SqlOperatorNotEqualsWithNullTest.java]: linebreak

File: src/test/java/com/axibase/tsd/api/method/sql/operator/SqlOperatorNotEqualsWithNullTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+     */
+    @Test
+    public void testIgnoringNullObjectsComparison1() {
+        String sqlQuery = String.format(
+                "SELECT value FROM '%s'%nWHERE tags.a != 'a'",
+                TEST_METRIC_NAME
+        );
+
+        Response response = executeQuery(sqlQuery);
+
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/operator/SqlOperatorNotEqualsWithNullTest.java]: linebreak

**Comment by Sild** [comment]
review done

**Comment by rmakulov** [comment]
lgtm

**Pull request merged**